package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "install": "node install.js",
     "test": "tape test/*.js",
-    "cache-clean": "rimraf ~/.electron"
```

Comment:
thats not a typo, its https://github.com/isaacs/rimraf. `rm` isn't cross platform, `rimraf` is